### PR TITLE
Fix: update the code-server and annotation

### PR DESCRIPTION
**Pull request by harshad16**
Fix: update the code-server and annotation

### CODE REVIEW

File: manifests/base/jupyter-minimal-gpu-notebook-imagestream.yaml [hunk 1]
```diff
@@ -18,13 +18,8 @@ spec:
     # N Version of the image
     - annotations:
         opendatahub.io/notebook-software: '[{"name":"CUDA","version":"11.8"},{"name":"Python","version":"v3.9"}]'
-<<<<<<< HEAD
-        opendatahub.io/notebook-python-dependencies: '[{"name":"JupyterLab","version":"3.6"},{"name":"Notebook","version":"6.5"}]'
-        openshift.io/imported-from: quay.io/modh/cuda-notebooks
-=======
         opendatahub.io/notebook-python-dependencies: '[{"name":"JupyterLab","version":"3.5"},{"name": "Notebook","version": "6.5"}]'
```
Comment (atheo89) [inline manifests/base/jupyter-minimal-gpu-notebook-imagestream.yaml:21]:
```suggestion
        opendatahub.io/notebook-python-dependencies: '[{"name":"JupyterLab","version":"3.6"},{"name": "Notebook","version": "6.5"}]'
```

File: manifests/base/kustomization.yaml [hunk 2]
```diff
@@ -9,7 +9,6 @@ resources:
   - jupyter-pytorch-notebook-imagestream.yaml
   - jupyter-tensorflow-notebook-imagestream.yaml
   - jupyter-trustyai-notebook-imagestream.yaml
-  - jupyter-habana-notebook-imagestream.yaml
```
Comment (atheo89) [inline manifests/base/kustomization.yaml:12]: Probably this removed accidentaly

**Review by atheo89 — COMMENTED** [review]
Thanks for opening this PR Overall looks good, however, I found out some small details to be fixed. 

Adding a note here: 
After merging this we shall update the SHA of the images on the `params.env` and `commits.env` files via digest updater

File: manifests/base/kustomization.yaml (hunk 2 above)
Comment (harshad16) [reply manifests/base/kustomization.yaml:12]: no it wasnt, it is present on line 7 already.

**Pull request merged**